inference-engine/ie_bridges/c/include/c_api/ie_c_api.h

```diff
@@ -709,6 +709,16 @@ INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode) ie_network_get_input_precision
  */
 INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode) ie_network_set_input_precision(ie_network_t *network, const char *input_name, const precision_e p);
 
+/**
+ * @brief Sets new batch size limit for the network.
+ * This function should be called before loading the network to the device.
+ * @ingroup Network
+ * @param network A pointer to ie_network_t instance.
+ * @param batch_size New batch size to be used.
+ * @return Status code of the operation: OK(0) for success.
+ */
+INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode) ie_network_set_batch_size(ie_network_t *network, size_t batch_size);
```

Comment:
Can we add new tests for this method in order to check that it works as expected?

Reply:
Added tests in a manner similar to `ie_infer_request_set_batch`. May you please explain, how can I validate these tests? Do they run automatically in CI for this commit?

Reply:
As I know, our precomit should include these tests.